cli/Cargo.toml

```diff
@@ -15,69 +15,69 @@ futures = { version = "0.3.1", features = ["compat"] }
 serde = { version = "1.0.102", features = ["derive"] }
 structopt = "0.3.8"
 
-sc-authority-discovery = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sc-chain-spec = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sc-cli = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain", features = ["wasmtime"] }
-sp-core = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sc-executor = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain", features = ["wasmtime"] }
-sc-keystore = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sc-service = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain", features = ["wasmtime"] }
-sp-inherents = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sp-keystore = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sc-transaction-pool = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sp-transaction-pool = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sp-authority-discovery = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sc-consensus-babe = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain", features = ["test-helpers"] }
-sp-consensus-babe = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sc-consensus = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sc-consensus-epochs = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sp-consensus = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sc-network = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sc-finality-grandpa = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sp-finality-grandpa = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sc-client-api = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
-sp-runtime = { rev = "2.0.5", git = "https://github.com/plugblockchain/plug-blockchain" }
+sc-authority-discovery = { branch = "feat/multi-currency-reserve", git = "https://github.com/plugblockchain/plug-blockchain" }
```

Comment:
Depending on the status of a branch is not a good idea. I would suggest releasing a tag on Plug like 2.0.6 for this purpose.